homework/src/Contributor.js

```diff
@@ -14,5 +14,12 @@ class Contributor {
   */
   render(contributorList) {
     // Replace this comment with your code
+
+    const div3 = Util.createAndAppend('div', contributorList, { 'class': 'container' });
```

Comment:
The name `div3` seems arbitrary in this context. There are no other `div` variables here that would warrant adding the suffix `3`.

But in fact, this should not be a `div` but an `li`.

Note also that you do not need to enclose object property names in quotes, unless those property names include characters that are not permitted in JavaScript variable names.